config/routes.rb

```diff
@@ -2,5 +2,8 @@
 root 'photos#index'
 devise_for :users
 resources :photos
+resources :users
+get 'user-pictures' => 'photos#user_pictures'
```

Comment:
This is not a restful route. You'll need to tear this down and use a restful route.